### DISCUSSION TITLE
✅ 🔗 Off Canvas Widget (Duplicate of #6225, #21769) - [ED-13937]

### DISCUSSION THREAD

**Discussion by rblackfox** [elementor/elementor#25770] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Off canvas allows users to create flout menus content panels, options panels and more without having to use a navigation menu.
Ultimate Elementor Add Ons has this already has this widget, but we use it in every build it should be in Elementor core.
https://ultimateelementor.com/widgets/off-canvas/

### Describe the solution you'd like.

Without having to use another navigation widget or external popup which might confuse disabled uses with unnecessary aria labels, an off canvas widget would make Elementor seem more "app like" and allow creators to stuff as many option panels or off screen content in their pages as they want 

### Describe alternatives you've considered.

Secondary navigation widgets and popups work, but are more difficult to create and I would imagine have sores SEO instead of an in-page off canvas widget.

### Additional context

Here is an example of an off canvas widget:
https://ultimateelementor.com/widgets/off-canvas/

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by loucasl** [comment]
This was actually just announced as a new widget coming soon in Elementor's Q2 roadmap event today!

https://www.youtube.com/live/A0NEw9Fz5eE?si=ZSmBJtSTFgpHEp5J

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
